scripts/interactions.js

```diff
@@ -285,13 +285,24 @@ damasflow_ondrop = function ( e )
 	if (path.indexOf('file://') === 0)
 	{
 		path = path.replace('file://', '');
+		var newPath=path;
 		var workdir=wd.concat(JSON.parse(localStorage["workdirs"]));
 		workdir.sort(function(a, b){
 			return b.length - a.length;
 		});
-		console.log(workdir);
-		for(var w=0;w<workdir.length;w++)
-			path= path.replace(new RegExp("^"+workdir[w]), '');
+		for(var w=0;w<workdir.length;w++){
+			newPath= path.replace(new RegExp("^"+workdir[w]), '');
```

Comment:
if I have workdirs = ["/a","/b","/c"]
and path = "/a/b/c/file"
then it will replace successively workdirs in path and the result will be "/file" but it is not what I want. correct?